arithmetization/src/main/java/net/consensys/linea/sequencer/txselection/LineaTransactionSelectorCliOptions.java

```diff
@@ -22,9 +22,10 @@
 public class LineaTransactionSelectorCliOptions {
   public static final int DEFAULT_MAX_BLOCK_CALLDATA_SIZE = 70000;
   private static final String DEFAULT_MODULE_LIMIT_FILE_PATH = "moduleLimitFile.json";
-
+  public static final long DEFAULT_MAX_GAS_PER_BLOCK = Long.MAX_VALUE;
```

Comment:
could we put here the expected default for Linea? If I am not wrong it should be 30M